pkg/acsengine/defaults.go

```diff
@@ -422,11 +429,11 @@ func setOrchestratorDefaults(cs *api.ContainerService, isUpdate bool) {
 			a.OrchestratorProfile.KubernetesConfig.UseInstanceMetadata = helpers.PointerToBool(api.DefaultUseInstanceMetadata)
 		}
 
-		if a.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == "" {
+		if !a.HasAvailabilityZones() && a.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == "" {
```

Comment:
Is `LoadBalancerSku` not used if the cluster doesn't have Availability Zones?

Reply:
if we are not using availability zones, then `LoadBalancerSku` should be set to `Basic` if not set.

Reply:
Oh I misread that, missed the `!`, makes sense 👍